plugin.php

```diff
@@ -59,6 +59,18 @@ function should_prevent_anonymous_access( WP_REST_Server $server, WP_REST_Reques
 		$settings = [];
 	}
 
+	/**
+	 * Allow OPTIONS requests to bypass the REST API Guard checks by default.
```

Comment:
Both this description and the `@param` documentation a few lines down seem to describe the effect of the filter, but not how to interact with it.

```suggestion
	 * Filters whether the REST API Guard should check OPTIONS requests.
```